src/mono/sample/wasm/browser/Wasm.Browser.Sample.csproj

```diff
@@ -7,4 +7,16 @@
   <ItemGroup>
     <WasmExtraFilesToDeploy Include="index.html" />
   </ItemGroup>
+
+  <PropertyGroup>
+    <_SampleProject>Wasm.Browser.Sample.csproj</_SampleProject>
+  </PropertyGroup>
+
+  <Target Name="CheckServe">
+    <Exec Command="dotnet tool install -g dotnet-serve" IgnoreExitCode="true" />
+  </Target>
+
+  <Target Name="RunSample" DependsOnTargets="BuildSampleInTree;CheckServe">
```

Comment:
You could move the two `RunSample` targets to `Directory.Build.*` as `RunSampleWithV8`, and `RunSampleWithBrowser`. And have `RunSample` in the sample projects, which depends on either of the first two targets.
This would mirror what we have in `wasm.mk`.